recipes/libevent/2.1.11/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+from conans import ConanFile, CMake
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def imports(self):
```

Comment:
Are these imports needed to run the example in test package? Is this because of openssl shared?

Reply:
Yes, as pointed in a review above, it is needed for resolving shared library dependencies.